CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
```

Comment:
If possible we should follow this CHANGELOG standard format: https://keepachangelog.com/en/1.1.0/

Reply:
DONE